.github/workflows/README.md

```diff
@@ -12,6 +13,72 @@ The following workflows can be found here:
 - [Java Gradle Plugin](https://github.com/bakdata/ci-templates/tree/main/.github/workflows#java-gradle-plugin)
 - [Java Gradle Release](https://github.com/bakdata/ci-templates/tree/main/.github/workflows#java-gradle-release)
 
+## Docker Build and Publish
+
+This workflow will use a Dockerfile to build and push images to any container registry.
+
+### Prerequisites
+
+This workflow requires a Dockerfile located in the repository.
+
+### Dependencies
+
+This workflow is built from multiple composite actions listed below:
+
+- [docker-build](https://github.com/bakdata/ci-templates/tree/main/actions/docker-build)
+- [docker-publish](https://github.com/bakdata/ci-templates/tree/main/actions/docker-publish)
+- [helm-package](https://github.com/bakdata/ci-templates/tree/main/actions/helm-package)
+- [commit-and-push](https://github.com/bakdata/ci-templates/tree/main/actions/commit-and-push)
```

Comment:
Where are you using the `helm-package` and `commit-and-push` actions in your workflow?